src/Tribe/Template.php

```diff
@@ -502,13 +502,27 @@ public function get_template_file( $name ) {
 	 * @return string|false Either the final content HTML or `false` if no template could be found.
 	 */
 	public function template( $name, $context = array(), $echo = true ) {
-		// If name is String make it an Array
-		if ( is_string( $name ) ) {
-			$name = (array) explode( '/', $name );
+		static $file_exists    = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 5314c170a472fa3bfad51b52e1ed2cda</sub>